diff -Nur old/iptables/libipt_p2p.c new/iptables/libipt_p2p.c
--- old/iptables/libipt_p2p.c	2004-02-29 00:53:13.000000000 +0000
+++ new/iptables/libipt_p2p.c	2005-04-12 21:06:00.000000000 +0000
@@ -68,18 +68,18 @@
 
 static struct iptables_match p2p =
 {
-    NULL,
-    "p2p",
-    IPTABLES_VERSION,
-    IPT_ALIGN(sizeof(struct ipt_p2p_info)),
-    IPT_ALIGN(sizeof(struct ipt_p2p_info)),
-    &help,
-    &init,
-    &parse,
-    &final_check,
-    &print,
-    &save,
-    opts
+    .next=NULL,
+    .name="p2p",
+    .version=IPTABLES_VERSION,
+    .size=IPT_ALIGN(sizeof(struct ipt_p2p_info)),
+    .userspacesize=IPT_ALIGN(sizeof(struct ipt_p2p_info)),
+    .help=&help,
+    .init=&init,
+    .parse=&parse,
+    .final_check=&final_check,
+    .print=&print,
+    .save=&save,
+    .extra_opts=opts
 };
 
 /*****************************************************************************/
